python/lsst/ts/observatory/control/base_camera.py

```diff
@@ -0,0 +1,533 @@
+# This file is part of ts_observatory_control.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["BaseCamera"]
+
+import abc
+import asyncio
+
+import numpy as np
+import astropy
+
+from . import RemoteGroup
+
+
+class BaseCamera(RemoteGroup, metaclass=abc.ABCMeta):
+    """Base class for camera instruments.
+
+    Parameters
+    ----------
+    components : `list` [`str`]
+        A list of strings with the names of the SAL components that are part
+        of the camera group.
+    instrument_setup_attributes : `list` [`str`]
+        Names of the attributes needed to setup the instrument for taking an
+        exposure. This is used to check for bad input with functions calls.
+    domain : `lsst.ts.salobj.Domain`
+        Domain for remotes. If `None` create a domain.
+    log : `logging.Logger`
+        Optional logging class to be used for logging operations. If `None`,
+        creates a new logger. Useful to use in salobj.BaseScript and allow
+        logging in the class use the script logging.
```

Comment:
In line 44 you use "create" and in line 47 "creates". Please choose one and stick to that. Also, I don't understand the last sentence.

Reply:
ok, I removed that last sentence and added some information on the documentation.